src/linear_algebra/matrix/determinant.lean

```diff
@@ -489,10 +489,11 @@ begin
   { intros σ _,
```

Comment:
I suspect this whole proof can be golfed using `fintype.sum_bijective` instead of `finset.sum_bij`. I think there's an even better lemma specific to equivs somewhere

Reply:
You are probably right: I just went for the quick fix!